sdk/appconfiguration/app-configuration/test/testHelpers.ts

```diff
@@ -70,3 +75,17 @@ export function assertEqualSettings(
   });
   assert.deepEqual(expected, actual);
 }
+
+export async function assertThrowsRestError(testFunction: () => Promise<any>, expectedStatusCode: number, message: string) : Promise<void> {
```

Comment:
you can use `assert.throws` for this, I believe: https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message

Reply:
I _almost_ can - the issue is the underlying function is async.

There is assert.rejects() in node 10 but our minimum platform is 8.x Perhaps that shouldn't matter for testing but I wasn't sure.

Reply:
ah boo, never mind then! 👍